targets/ingester/src/cli.js

```diff
@@ -159,21 +185,32 @@ async function main() {
     console.log(
       `ready to ingest ${documents.length} documents from ${pkgName}`
     );
-    const inserts = await batchPromises(documents, insertDocument, 10);
+    const chunks = chunk(documents, 100);
+    const inserts = await batchPromises(chunks, insertDocuments, 10);
```

Comment:
Juste pour être sûr de bien comprendre: on découpe les documents en chunks de 100, et ensuite on fait des batchs de 10 chunks en parallèle jusqu'à épuisement des chunks ? Il y a une raison à ce 100 ?

Reply:
je l'aurais appellée "splitIntoChunks" la fonction il manque un petit verbe à mon avis

Reply:
Le truc qui m'a fait bugguer c'est que dans batchPromise le premier param est un tableau de tableau et que le handler fait un map, mais c'est très smart !
Et chunk ça va très bien en fait c'est un verbe aussi, c'est juste que je suis mauvais en anglais 👌

Reply:
on lance 10 requêtes en parallèle et dans chaque envoi, on met  100 documents, (contre 1 document avant) histoire  que l'exécution ne prenne pas trop de temps je vais faire un test 1 vs 50 vs 100 vs 500

Reply:
x1 Finish ingest 12073 documents
✨  Done in 370.11s.

x10 Finish ingest 12073 documents
✨  Done in 179.28s.

x50 Finish ingest 12073 documents
✨  Done in 111.18s.

x100 Finish ingest 12073 documents
✨  Done in 112.22s.

x300 Finish ingest 12073 documents
✨  Done in 110.48s.

j'ai testé de doublé le nombre de thread aussi (chunck 50 ou chenuk 100) et je plafonne autour  110s

Reply:
yes je pensas plus à un truc comme 500 et 1000 mais c'est kif-kif !